src/js/views/settingsView.js

```diff
@@ -0,0 +1,124 @@
+var Marionette = require('backbone.marionette'),
+    _ = require('underscore'),
+    $ = require('jquery'),
+    bindAll = require('lodash.bindall');
```

Comment:
I keep them separated for a reason, some sort of convention.

Usually I specify my requires this way:

1. One block for third-parties (backbone, underscore, jquery, etc.)
1. One block for local utils/global stuff (analytics, logging, notifications, etc.)
1. One block for views
1. One block for controllers
1. One block for models
1. One block for templates

IMO it makes it easier to read the dependencies a module has.